src/Response.php

```diff
@@ -118,9 +125,10 @@ class Response
      * @param \DateTimeImmutable $thisUpdate
      * @param string $certificateSerialNumber
      */
-    protected function __construct(DateTimeImmutable $thisUpdate, $certificateSerialNumber)
+    protected function __construct(DateTimeImmutable $thisUpdate, ?DateTimeImmutable $nextUpdate, $certificateSerialNumber)
```

Comment:
As per `composer.json`, this library must be compatible with PHP 5.5.9...

Reply:
Urgh. Would you prefer phpdoc only?

Or a composer.json update.

Reply:
The problem is `?DateTimeImmutable`, which requires PHP 7.1+.
The compatible way to write it is:

```php
protected function __construct(DateTimeImmutable $thisUpdate, $certificateSerialNumber,  DateTimeImmutable $nextUpdate = null)
```